install/install.go

```diff
@@ -115,6 +115,34 @@ func checkPods(c *k8s.Client) int {
 	return 0
 }
 
+func checkTerminatingPods(c *k8s.Client) int {
+	cursor := [4]string{"|", "/", "—", "\\"}
+	fmt.Printf("🔴   Checking if KubeArmor pods are stopped ...")
+	stime := time.Now()
+	otime := stime.Add(600 * time.Second)
+	for {
+		time.Sleep(200 * time.Millisecond)
+		pods, _ := c.K8sClientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{LabelSelector: "kubearmor-app", FieldSelector: "status.phase=Running"})
```

Comment:
Does all the kubearmor related pods have this specific label ?
Also please correct me if I am wrong, I see only one value **kubearmor-app**, does it not expected to have key:value in labels ?

Reply:
@seswarrajan Yes label `kubearmor-app` is present in all the pods created by `kubeArmor` with different values like `kubearmor-app=kubearmor-annotation-manager`, `kubearmor-app=kubearmor-host-policy-manager`, `kubearmor-app=kubearmor-policy-manager` and `kubearmor-app=kubearmor-relay` So i think it's wiser move to only consider key `kubearmor-app` to filter pods.